molgenis/molgenis-service-armadillo
# feat: M10506 UI create link file

how to test:
- go to localhost in the browser
- go to projects tab
- select lifecycle
- select a table
- press create view button
- fill in all fields
- see a new table is created in the selected destination
- go to another project
- create or select a folder
- click "create table to link from..."
- fill in fields, create view
- refresh
- see view is in destination

todo:
